lib/ex_cell/base.ex

```diff
@@ -1,49 +1,11 @@
 defmodule ExCell.Base do
   @moduledoc false
```

Comment:
Don't we want docs for this? Or just laziness? :P

Reply:
See previous comments